HW03/Alexey_Zalyotov/lib/student.rb

```diff
@@ -0,0 +1,51 @@
+class Student < Person
+  OPEN = :open
+  IN_PROGRESS = :in_progress
+  SUBMITTED = :submitted
+  CLOSED = :closed
+
+  def initialize(name:, surname:)
+    super
+    @active_task = nil
+  end
+
+  def subscribe_repo(repo)
+    repo.students << self
+  end
+
+  def take_task(repo)
+    repo.tasks.each do |task|
+      next if task.status != OPEN
```

Comment:
can we move all these constants to the `Task` class? Because it's not the responsibility of Student to know about it 
```suggestion
      next if task.status != Task::OPEN
```

Reply:
I didn’t know that we could do that. Thank you